deployer/prepare.sh

```diff
@@ -30,6 +30,13 @@ shuttle-tide = { path = "/usr/src/shuttle/services/shuttle-tide" }
 shuttle-tower = { path = "/usr/src/shuttle/services/shuttle-tower" }
 shuttle-warp = { path = "/usr/src/shuttle/services/shuttle-warp" }' > $CARGO_HOME/config.toml
 
+# Install protoc since some users may need it
```

Comment:
We used to install `protoc` in the Containerfile, but we don't need it for our crates anymore. However, removing it in the deployer may be an inadvertent breaking change, as some users may rely on it. So I install it here instead.